### DISCUSSION TITLE
Equations for pediatric growth

### DISCUSSION THREAD

**Discussion by MartjeVanNeste** [Open-Systems-Pharmacology/Forum#1711] [Questions & Problems]
Hi,

I am working on a data set for infants and wanted to compare the data that I collected with the equations in PK-Sim. However, I don't seem to find the equations used. For the European population, the ICRP publication mentions the height and mass of infants at 6 time points but I can't find an equation for a height / weight curve.
Would it be possible to share the height and weight equation used in PK-Sim?

Thank you very much!

**Comment by wilbertdew** [comment]
Hi @MartjeVanNeste, I believe most of those changes are incorporated in the database that you can find here https://github.com/Open-Systems-Pharmacology/PK-Sim/tree/develop/src/Db (file PKSimDB.sqlite). You can access this file with the DB browser for SQLite, but you can also access it directly in R with the right package (e.g. "RSQLite"). I believe the tables of interest are "VIEW_PARAMETER_DISTRIBUTIONS" and "VIEW_ONTOGENIES". It might be a bit of work but maybe there are more convenient approaches to find this @Yuri05 ?

**Comment by Yuri05** [comment]
@MartjeVanNeste As @wilbertdew said, the data is stored in the PK-Sim database (s. the [database documentation](https://github.com/Open-Systems-Pharmacology/developer-docs/blob/main/PKSim-DataBase/DB.md) for details, particularly the description for the storage of [age dependent data points](https://github.com/Open-Systems-Pharmacology/developer-docs/blob/main/PKSim-DataBase/DB.md#tab_container_parameter_curves). When creating an individual in PK-Sim, this data is further transformed based on the input weight/height etc. according to the ["Create Individual Algorithm"](https://github.com/Open-Systems-Pharmacology/OSPSuite.Documentation/wiki/Create-Individual-Algorithm)
